docs/src/troubleshooting.md

```diff
@@ -119,93 +122,95 @@ kubectl exec -i new-cluster-example-1 -c postgres \
 ```
 
 !!! Important
-    The example in this section assumes that you have no other global objects
-    (databases and roles) to dump and restore, as per our recommendation. In case
+    This example assumes that you have no other global objects
+    (databases and roles) to dump and restore, as per our recommendation. If
     you have multiple roles, make sure you have taken a backup using `pg_dumpall -g`
-    and you manually restore them in the new cluster. In case you have multiple
+    and you manually restore them in the new cluster. If you have multiple
     databases, you need to repeat the above operation one database at a time, making
-    sure you assign the right ownership. If you are not familiar with PostgreSQL,
-    we advise that you do these critical operations under the guidance of
+    sure you assign the right ownership. If you're not familiar with PostgreSQL,
+    we recommend that you do these critical operations under the guidance of
     a professional support company.
```

Comment:
I can't follow in this note what "above" operation is being referred to. Also, would it make sense to move this note up so it's understood before learning about the commands?